client/signup/steps/plans/index.jsx

```diff
@@ -190,6 +191,40 @@ export class PlansStep extends Component {
 		return subHeaderText || translate( 'Choose a plan. Upgrade as you grow.' );
 	}
 
+	getSubHeaderTextForExperiment() {
+		const { hideFreePlan, flowName, translate } = this.props;
+		const defaultSubHeaderText = this.getSubHeaderText();
+
+		if ( ! isDesktop() || 'onboarding' !== flowName ) {
```

Comment:
I can't manage to see the variation text defined below using the ExPlat bookmarklet. I suppose it's because `! isDesktop()` here. Can you help me confirm if it should be the other way around? 🤔